canonical/juju-dashboard
# WD-7718 - feat: Remove additional controllers feature

## Done

- Removed additional controllers feature.

## QA

- Verify that "Register a controller" button does not appear in controllers page.
- Verify that `/controllers?panel=register-controller` route doesn't open the register controller panel.
- Verify that artifacts related to additional controller feature have been removed.

## Details

- https://warthogs.atlassian.net/browse/WD-7718

## Screenshots

![Screenshot from 2023-12-06 05-52-29](https://github.com/canonical/juju-dashboard/assets/108150922/75391b12-b792-447d-bf7c-596280b14def)
